docs/admin/performance/selects.rst

```diff
@@ -107,6 +107,93 @@ to the original result.
    read from the file system. This makes it an ideal candidate for modulo
    based sampling.
 
+
+.. _rewrite-join-as-cte:
+
+Rewrite JOINs as CTEs
+=====================
+
+The article about `using common table expressions to speed up queries`_ shares
+a pattern you can use to replace JOINs with CTEs in your SQL queries, in order
+to achieve consistent and faster execution times.
+
+**Example**
```

Comment:
What CTEs to use depends on the profile of the data, should we add a caution note about this?